### PR TITLE
Update Getting started topics for style, syntax, layout, spacing

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

This PR updates the Install and Getting started topics, focusing on applicaiton of style, layout, spacing, and overall consistency.

Changes:
- Use semantic line breaks instead of 80 column
- Use lazy numbering for steps
- Remove empty spaces at the end of lines
- Fix cases of `Flow` that were missed in the conversion to a cascading variable
- Rework text to remove most of the passive voice
- Update the placeholder variables to match current Grafana Writer's Toolkit style
- Correct spelling errors
- Tidy up grammar and style used throughout the tutorials and try to align the language used in all tutorials (it's not perfect, but it's closer now)
- Clean up and standardise the `Replace the following` text for placeholder variables
- Clean up some of the step nesting - some steps were nested too deeply, and step number references no longer were valid due to changes in the website and the way numbering works.
- Fix and standardise cross reference linking (use `doc/reference` where possible)

#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/flow/getting-started/migrating-from-static.md [hunk 1]
```diff
@@ -36,118 +36,116 @@ This topic describes how to:
...
+
+1. [Start][] {{< param "PRODUCT_NAME" >}} using the new {{< param "PRODUCT_NAME" >}} configuration from _`<OUTPUT_CONFIG_PATH>`_:
 
 ### Debugging
 
-1. If the convert command cannot convert a [Static] configuration, diagnostic
-   information is sent to `stderr`. You can use the `--bypass-errors` flag to
-   bypass any non-critical issues and output the {{< param "PRODUCT_NAME" >}} configuration
-   using a best-effort conversion.
+1. If the convert command cannot convert a [Static] configuration, diagnostic information is sent to `stderr`.
```
Comment (ptodev) [inline docs/sources/flow/getting-started/migrating-from-static.md:68]: Not sure why `[Static]` isn't `[Static][]`? It seems to work though.

File: docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md [hunk 2]
```diff
@@ -34,17 +34,15 @@ This topic describes how to:
...
+* Ensure that you have basic familiarity with instrumenting applications with OpenTelemetry.
+* Have a set of OpenTelemetry applications ready to push telemetry data to {{< param "PRODUCT_NAME" >}}.
 * Identify where {{< param "PRODUCT_NAME" >}} will write received telemetry data.
 * Be familiar with the concept of [Components][] in {{< param "PRODUCT_NAME" >}}.
 * Complete the [Collect open telemetry data][] getting started guide. You will pick up from where that guide ended.
 
 ## The pipeline
 
-You can start with the {{< param "PRODUCT_NAME" >}} configuration you created in the previous getting started guide:
+You can start with the {{< param "PRODUCT_NAME" >}} configuration you created in the previous Getting Started guide.
```
Comment (ptodev) [inline docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md:45]:
```suggestion
You can start with the {{< param "PRODUCT_NAME" >}} configuration you created in the [OpenTelemetry to Grafana stack][] Getting Started guide.
```
I think this should be a link to the guide that we are referring to.

**Review by ptodev — APPROVED** [review]
Thanks! It does look much more polished with these changes.

File: docs/sources/flow/getting-started/collect-opentelemetry-data.md [hunk 3]
```diff
@@ -31,89 +31,76 @@ This topic describes how to:
...
-        3. Replace `PASSWORD` with the basic authentication password or API key to
-           use.
+       - _`<BASIC_AUTH_LABEL>`_: The label for the component, such as `default`.
+         The label you use must be unique across all `otelcol.auth.basic` components in the same configuration file.
+       - _`<USERNAME>`_: The basic authentication username.
+       - _`<PASSWORD>`_: The basic authentication password or API key.
 
-    2. Add the following line inside of the `client` block of your
-       `otelcol.exporter.otlp` component:
+    1. Add the following line inside of the `client` block of your `otelcol.exporter.otlp` component:
```
[the remark this answers is not in the document]
Comment (clayton-cornell) [reply docs/sources/flow/getting-started/collect-opentelemetry-data.md:87]: Yes, it renders correctly. Markdown doesn't care what the actual number is as long as the first number in a list is a 1.

You could have:
```
1. Text
5. Text
8. Text
2. Text
3. Text
2. Text
```
And the list would render in order as 1, 2, 3, 4 etc.
https://www.markdownguide.org/basic-syntax/#lists-1

So we've opted to encourage using "lazy numbering"

File: docs/sources/flow/getting-started/migrating-from-static.md (hunk 1 above)
Comment (clayton-cornell) [reply docs/sources/flow/getting-started/migrating-from-static.md:68]: Fixed in the next commit.

File: docs/sources/flow/getting-started/migrating-from-operator.md [hunk 4]
```diff
@@ -257,10 +262,47 @@ loki.write "loki" {
...
+- _`<LOKI_URL>`_: The endpoint of your Loki instance.
 
+The logging subsystem is very powerful and has many options for processing logs. For further details, see the [component documentation][].
 
 ## Integrations
 
-The `Integration` CRD is not supported with {{< param "PRODUCT_NAME" >}}, however all static mode integrations have an equivalent component in the [`prometheus.exporter`](https://grafana.com/docs/agent/latest/flow/reference/components) namespace. The reference docs should help convert those integrations to their Flow equivalent.
+The `Integration` CRD isn't supported with {{< param "PRODUCT_NAME" >}}.
+However, all static mode integrations have an equivalent component in the [`prometheus.exporter`][] namespace.
+The reference docs should help convert those integrations to their {{< param "PRODUCT_NAME" >}} equivalent.
```
[the remark this answers is not in the document]
Comment (clayton-cornell) [reply docs/sources/flow/getting-started/migrating-from-operator.md:275]: Fixed in the next commit.

File: docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md (hunk 2 above)
Comment (clayton-cornell) [reply docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md:45]: This should probably be `[Collect open telemetry data][]` instead :-)

Fixed in the next commit.

File: docs/sources/flow/getting-started/collect-opentelemetry-data.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (clayton-cornell) [reply docs/sources/flow/getting-started/collect-opentelemetry-data.md:87]: As long as each element is indented properly, it should work as expected.

**Pull request merged**